knowledge-base/bind-sortdescriptors-collectionview-net-maui.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Binding SortDescriptors in CollectionView for .NET MAUI
+description: Learn how to use data binding with CollectionView SortDescriptors in .NET MAUI applications.
+type: how-to
+page_title: How to Bind SortDescriptors in .NET MAUI CollectionView
+slug: bind-sortdescriptors-collectionview-net-maui
+tags: collectionview, sortdescriptors, databinding, .net maui, observablecollection
+res_type: kb
+ticketid: 1677846
+---
+
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 9.0.0 | Telerik UI for .NET MAUI CollectionView | [Dobrinka Yordanova](https://www.telerik.com/blogs/author/dobrinka-yordanova) | 
+
+## Description
+
+In my application, I want to use data binding to bind to an ObservableCollection of sort descriptors.
+
+This knowledge base article also answers the following questions:
+- How to dynamically sort items in a CollectionView using SortDescriptors in .NET MAUI?
+- How to bind SortDescriptors to a ViewModel property in .NET MAUI?
+- How to update CollectionView sorting based on user input in .NET MAUI?
+
+## Solution
+
+To bind the `SortDescriptors` collection in a `CollectionView` to a property in the ViewModel, follow these steps:
```

Comment:
When referring to the view model as a general concept, write it as two words. When referring to a specific `ViewModel` in a project, then use the PascalCase variant in monospace.
```suggestion
To bind the `SortDescriptors` collection in a CollectionView to a property in the view model, follow these steps:
```